app/_landing_pages/insomnia/documents.yaml

```diff
@@ -0,0 +1,239 @@
+              alt_text: Image of a Document. 
+  
+  - header:
+      type: h2
+      text: "Design API specs"
+    columns:
+      - blocks:
+          - type: image
+            config:
+              url: /assets/images/insomnia/document-spec.png
+              alt_text: Image of API spec in a document.
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: | 
+                    Using Insomnia you can create or upload an API spec to a document. A preview of your spec is generated, which you can view in the right pane.
+                    
+                    All specifications imported into Insomnia are linted, and error messages display at the bottom of the center panel. 
+                    Error details include the error type, line number, and error message.
+
+                    In the left-side panel, a menu populates with information about your API spec. 
+          - type: button
+            config:
+              align: left
+              text: Learn more about designing APIs
+              url: /insomnia/design/
+  - header:
+      type: h3
+      text: "Get started"
+    columns:
+      - blocks:
+        - type: card
+          config:
+            title: Create a design document
+            cta:
+              text: Learn more
+              url: /how-to/create-a-design-document/
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Import an API specification
+            cta:
+              text: Learn more
+              url: /how-to/import-an-api-spec-as-a-document/
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Lint a design document
+            cta:
+              text: Learn more
+              url: /
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Automate design document linting
+            cta:
+              text: Learn more
+              url: /
+              align: end
+  - header:
+      type: h2
+      text: "Debug API specs"
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: | 
+                    To debug your API spec, you can generate a collection of requests from it and debug those. 
+                    
+                    In the sidebar, you'll see a list of the requests. When you select one, you can send requests and then see the response in the right menu. The center panel will populate with the following:
+                - type: unordered_list
+                  items:
+                    - Request URL
+                    - Request type
+                    - Base URL variable (defined in the OpenAPI specification)
+                    - Request body (if applicable)
```

Comment:
I think having both request URL and base URL i might be confusing. I would say base URL, endpoints, and query parameters. And the list is missing request headers.